api/base/exceptions.py

```diff
@@ -1,11 +1,19 @@
 import httplib as http
 
 from rest_framework import status
-from rest_framework.exceptions import APIException, ParseError
+from rest_framework.exceptions import APIException
 
 def json_api_exception_handler(exc, context):
     """ Custom exception handler that returns errors object as an array """
 
+    # We're deliberately not stripping html from exception detail.
+    # This creates potential vulnerabilitys to script injection attacks
+    # when returning raw user input into error messages.
+    #
+    # Fortunately, Django's templating language strips markup bu default,
+    # but if our frontend changes we may lose that protection.
```

Comment:
To be clear: the frontend discussion refers to the DRF browsable API tool using django's templates, right?

Two typos here: 
1) "vulnerabilities"
2) "but", not "bu"